src/GetTodosWithUsers.js

```diff
@@ -0,0 +1,10 @@
+function GetTodosWithUsers(todos, users) {
```

Comment:
c маленькой буквы, это же не компонент
```suggestion
function getTodosWithUsers(todos, users) {
```